src/main/java/frc/util/SwerveModuleConfig.java

```diff
@@ -0,0 +1,120 @@
+package frc.util;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.TalonFXSensorCollection;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+import frc.robot.Constants;
+
+public class SwerveModuleConfig {
+    //Module Variables
+    private final int idDrive;
+    private final int idSteer;
+    private final double degOffSet;
+
+    //Drive objects for the Module
+    private final WPI_TalonFX drive;
+    private TalonFXSensorCollection driveEncoder;
+
+    //Steering objects for the Module (encoder will be analogInput OR canCoder)
+    public final WPI_TalonFX steer;
+    private TalonFXSensorCollection steerEncoder;
+
+    //PID Vars
+    private double kPSteer;
+    private double kISteer;
+    private double kDSteer;
+    private double kFSteer;
+    private double kPDrive;
+    private double kIDrive;
+    private double kDDrive;
+    private double kFDrive;
```

Comment:
We can get rid of this (since we never use it)

Reply:
Shreyas to do: Add Better Odometry bc my odometry sucs
Shorya and Lydia to do (Aaron Supervision) : Format, read, understand, find better way to configure PID.
Aaron to do: Finish tuner for swerve module